components/orchestration/internal/workflow/instance.go

```diff
@@ -15,7 +15,7 @@ type Instance struct {
 	UpdatedAt     time.Time  `json:"updatedAt"`
 	Terminated    bool       `json:"terminated"`
 	TerminatedAt  *time.Time `json:"terminatedAt,omitempty"`
-	Statuses      []Stage    `json:"status,omitempty" bun:"rel:has-many,join:id=instance_id"`
+	Statuses      []Stage    `json:"statuses,omitempty" bun:"rel:has-many,join:id=instance_id"`
```

Comment:
Can you keep "statuses" until feat/v2 is merged?
Maybe just set a TODO.
Because @reslene is actively working on orchestration, and it will lead to a breaking change we will forgot.